snuba/manual_jobs/__init__.py

```diff
@@ -0,0 +1,29 @@
+import os
+from abc import ABC, abstractmethod
+from typing import Any, cast
+
+from snuba.utils.registered_class import RegisteredClass, import_submodules_in_directory
+
+
+class Job(ABC, metaclass=RegisteredClass):
```

Comment:
I don't think the base class/interface belongs in __init__ here. 

Let's separate this out into snuba/manual_jobs/base.py or snuba/manual_jobs/job.py or something. In general `__init__` should not contain much (I do think it's reasonable enough for it to contain the import statement below, though).

Reply:
I guess we use this pattern with `ClickhouseQueryProcessor`. Maybe this can stay

Reply:
yeah, we can ignore this for now